tools/projmgr/docs/Manual/Overview.md

```diff
@@ -234,16 +227,21 @@ Print list of unresolved project dependencies. Device, board, and software compo
 `*.csolution.yml` and `*.cproject.yml` files. The list may be filtered by words provided with the option `--filter`:
 
 ```text
-csolution list dependencies -s mysolution.csolution.yml [-f "<filter words>"]
+csolution list dependencies mysolution.csolution.yml [-f "<filter words>"]
 ```
 
-#### Build Projects
+#### Create Build Information
 
-Convert `example.csolution.yml` into `*.cprj` file(s). This `*.cprj` file can be compiled using the `cbuild` command.
+Convert `example.csolution.yml` into build information files.
 
 ```text
 csolution convert -s example.csolution.yml
-cbuild example.debug.cprj
+```
```

Comment:
Please remove the `-s` in the string above.
csolution convert example.csolution.yml